Cargo.toml

```diff
@@ -110,7 +110,8 @@ sha2-asm = "0.5.3"
 
 [workspace]
 members = [
-    ".", 
+    ".",
+    "clarity",
```

Comment:
Not about this line but it looksl ike "unit tests" is failing.. Do you know why that would be?

```
failures:

---- clarity::test::test_assets stdout ----
initialize
INFO [1629087718.866954] [clarity/../src/vm/functions/mod.rs:445] [clarity::test::test_assets] "``... to be a completely separate network and separate block chain, yet share CPU power with Bitcoin`` - Satoshi Nakamoto"
check tokens
Error reading file: sample-contracts/tokens-ft.clar
Caused by: No such file or directory (os error 2)
thread 'clarity::test::test_assets' panicked at 'explicit panic', clarity/../src/clarity.rs:122:9

---- clarity::test::test_samples stdout ----
initialize
INFO [1629087720.500553] [clarity/../src/vm/functions/mod.rs:445] [clarity::test::test_samples] "``... to be a completely separate network and separate block chain, yet share CPU power with Bitcoin`` - Satoshi Nakamoto"
check tokens
Error reading file: sample-contracts/tokens.clar
Caused by: No such file or directory (os error 2)
thread 'clarity::test::test_samples' panicked at 'explicit panic', clarity/../src/clarity.rs:122:9

---- vm::docs::test::test_examples stdout ----
thread 'vm::docs::test::test_examples' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', clarity/../src/vm/docs/mod.rs:1967:77


failures:
    clarity::test::test_assets
    clarity::test::test_samples
    vm::docs::test::test_examples

test result: FAILED. 1218 passed; 3 failed; 94 ignored; 0 measured; 0 filtered out; finished in 922.25s

error: test failed, to rerun pass '-p clarity --lib'

Exited with code exit status 101

CircleCI received exit code 101

```

Reply:
looking into this...